feature-libs/asm/customer-360/components/sections/asm-customer-customer-coupon/asm-customer-customer-coupon.component.html

```diff
@@ -0,0 +1,25 @@
+<cx-asm-customer-promotion-listing
+  [headerText]="'customer360.customerCoupons.headerText' | cxTranslate"
+  [emptyStateText]="
+    'customer360.customerCoupons.emptyDescription' | cxTranslate
+  "
+  [applyButtonText]="
+    'customer360.customerCoupons.applyButtonText' | cxTranslate
+  "
+  [applied]="'customer360.customerCoupons.applied' | cxTranslate"
+  [removeButtonText]="
+    'customer360.customerCoupons.removeButtonText' | cxTranslate
+  "
+  [entries]="entries$ | async"
+  [showRemoveButton]="true"
+  [showApplyButton]="true"
+  [isCustomerCoupon]="true"
+  (refreshCustomerCouponList)="changeTab($event.assignable, $event.searchQuery)"
```

Comment:
not only changeTab, others like search will emit refreshCustomerCouponList , maybe function name can refine ?

Reply:
only click enter or the search button will trigger search.